code/game/turfs/open.dm

```diff
@@ -63,7 +64,16 @@
 				return 0
 			if(C.m_intent=="walk" && (lube&NO_SLIP_WHEN_WALKING))
 				return 0
-
+		if(istype(C, /mob/living/carbon/human))
+			var/mob/living/carbon/human/H = C
+			if(istype(H.shoes))
+				if(H.shoes.flags&NOSLIP && lube&SLIDE && !lube&GALOSHES_DONT_HELP)
+					var/olddir = C.dir
+					if(lube&SLIDE)
+						for(var/i=1, i<4, i++)
+							spawn (i)
+								step(C, olddir)
```

Comment:
This will cause the same issue as the old 'hulk not falling on lube but sliding' bug.

Make a row of slippery turf, get noslip shoes, walk on the first turf, and before the sliding stops, when you are still in the midle of the row, try to walk back. Result: you're crashing the game with lag, congratulation.